CRM/Activity/Import/Form/DataSource.php

```diff
@@ -31,20 +31,11 @@ public function buildQuickForm() {
     parent::buildQuickForm();
 
     // FIXME: This 'onDuplicate' form element is never used -- copy/paste error?
-    $duplicateOptions = [];
```

Comment:
Per the comment, this one is impossible to `r-run` because this form element is never added to the template -- it's completely unused. I recommend deleting this entire function.

Reply:
I was looking at that yesterday and yes it's unused now but all that would needed to make it used would be to add the field to the tpl and then actually use it in the parser subclass. It looks like it's supported to be passed on all the way thru, just not implemented at the end.

Reply:
My sense is is that if the other onDuplicate ones work then we can be relatively confident this one will as well

Reply:
I support ripping out any code we reasonably believe to be unused

Reply:
Ok well if someone could `r-run` the rest of this PR then we can merge it and making a decision one way or the other about Activity Import doesn't have to be a blocker.